src/components/HomePage/Grid/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styles from './styles.module.css'
+
+export default function Grid(props) {
+    
+
+    return (
+        
+        
+        <div className={styles["main-container"]}>
+
+            <h4 className={styles.title}>{props.title}</h4>
+        
```

Comment:
Is it me or a Github thing? Im seeing some white lines, a formatter conf file is a good convention, a backlog item is you get enough time :)